addon/templates/components/bootstrap-data-table.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="bootstrap-data-table-component" style="padding: 5px">
+  {{#if title}}
+    <div class="row">
+      <div class="col-md-12">
+        {{#if titleSmall}}
+          <p class="lead text-center">{{title}}</p>
+        {{else}}
+          <h3 class="text-center">{{title}}</h3>
+        {{/if}}
+      </div>
+    </div>
+  {{/if}}
+  <table class="table {{if striped 'table-striped'}} {{if responsive 'table-responsive'}} {{if bordered 'table-bordered'}} {{if canHover 'table-hover'}} {{if condensed 'table-condensed'}}" style="background-color: white">
```

Comment:
I think the responsive is not a class in the table, but a div wrapped around the table element.
```
<div class="table-responsive">
  <table class="table">
    ...
  </table>
</div>
```